packages/graphql-testing/src/graphql-controller.ts

```diff
@@ -33,14 +28,18 @@ export class GraphQL {
     mock: GraphQLMock | undefined,
     {unionOrIntersectionTypes = [], cacheOptions = {}}: Options = {},
   ) {
+    const possibleTypes = {};
+
+    unionOrIntersectionTypes.forEach((supertype) => {
```

Comment:
I wonder if we can add some more type safety. The tools we use to generate the the unions file almost always looks like this: https://github.com/Shopify/flow-app/blob/master/src/graphql/schema/fragmentTypes.json but would be nicer than `any[]` to have `UnionOrIntersection[]` and `PossibleType`.